build/rpm/main.go

```diff
@@ -206,6 +206,8 @@ URL:                https://kaia.io
 Source0:            %{name}-%{version}.tar.gz
 BuildRoot:          %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
 
+%global debug_package %{nil}
```

Comment:
Hi @blukat29 @yoomee1313 , I found that when we add this, it will ignore the debugsourcefiles.list and success to build the rpm. Is that okay?

Reply:
While I'm not familiar with RPM, it seems that debugsourcefiles.list is not particularly important in debugging Golang applications. So I think it's safe to exclude this file.

Reply:
I've tested and it is fine to ignore it.